src/base/types.ts

```diff
@@ -67,5 +67,9 @@ export const TYPES = {
     ICommandPaletteActionProviderRegistry: Symbol('ICommandPaletteActionProviderRegistry'),
     ICommandPaletteActionProvider: Symbol('ICommandPaletteActionProvider'),
     IEditLabelValidator: Symbol('IEditLabelValidator'),
-    IEditLabelValidationDecorator: Symbol('IEditLabelValidationDecorator')
+    IEditLabelValidationDecorator: Symbol('IEditLabelValidationDecorator'),
```

Comment:
Minor comment here: Could we keep the alphabetic order (ignoring leading I)

Reply:
Yes, I'll sort those in a sec.